.github/workflows/generate-integrations-docs.yml

```diff
@@ -0,0 +1,49 @@
+---
+  # CI workflow used to generate documentation from integrations/integrations.js.
+
+name: Generate Go Integrations Documentation
+on:
+  schedule:
+    - cron: '0 10 * * *'
+  workflow_dispatch: null
```

Comment:
Why cron? Should it be 
- pushes to master + 
- some files filter
 
?

Reply:
the ideal would be to track changes on the integrations.js, we don't want to change readmes when yaml files change, as the integrations.js will be the same :thinking: 

Can we track file changes from other repos?

Reply:
Ah, I see. As it is now this workflow should be triggered by the one in netdata/netdata (which in 99% cases will do nothing) or cron.

Reply:
Probably it is better to not add this workflow in netdata/go.d.plugin but have it in netdata/netdata and let it make a PR in netdata/go.d.plugin if needed. What do you think?

Reply:
The script atm is designed to run in both repos, I haven't tested it to run with go checked out in the netdata repo :thinking: 

How would we then PR the files over to here?

I thought the simplest would be to run it with cron, and then if we merge the integrations PR in the other repo, manually run it.

@Ferroin there is no trigger to refer to a merged PR on netdata/netdata eh?
If there was, we would use that

Reply:
For updating things in the agent repo, we should absolutely just run it in the same workflow that is generating the integrations.js file.

For updating things here, add a `workflow_dilspatch` trigger on this workflow, and I’ll add a workflow in the agent repo to trigger it whenever the `integrations.js` file changes on the master branch.